lib/kwalify_to_json_schema/converter.rb

```diff
@@ -87,19 +87,25 @@ def process(target, kelem, path = [])
           mapping.each_pair { |name, e|
             # Ignore mapping default value
             if name == "="
-              new_issue path, Limitations::MAPPING_DEFAULT_VALUE_NOT_SUPPORTED
-              next
+              process(target["additionalProperties"] = {}, e, path)
```

Comment:
The following situation will not be manage properly:
```
A: 10
B: 20
=: -1      # default value
```

First `process(target, kelem, path)` is expecting a `Hash` for `kelem`.
Secondly no issue will be reported to the user.

I suggest the following:

```suggestion
              if e.is_a?(Hash)
                process(target["additionalProperties"] = {}, e, path)
              else
                new_issue path, Limitations::ONLY_DEFAULT_RULE_SUPPORTED_FOR_DEFAULT_MAPPING
              end
```

`ONLY_DEFAULT_RULE_SUPPORTED_FOR_DEFAULT_MAPPING` to be created.

:information_source: No need to edit `README.md` manually:
* Edit `limitations.rb`
* Then run `rake doc:update`
* `README.md` will be updated using `README.md.erb` template